bin/build-for-publishing.js

```diff
@@ -87,7 +87,7 @@ Promise.resolve()
   .then(
     // eslint-disable-next-line
     () => console.log('build-for-publishing completed successfully!'),
-    error => {
+    (error) => {
```

Comment:
Can we change `arrowParens` back to `"avoid"`, if only just minimize the diff?

Reply:
Ah! Sure, I didn't realise it was still an option. I thought they removed it in v2

Reply:
Ok, done 👍 I was thinking of [Always add a space after the `function` keyword](https://prettier.io/blog/2020/03/21/2.0.0.html#always-add-a-space-after-the-function-keyword-3903httpsgithubcomprettierprettierpull3903-by-j-f1httpsgithubcomj-f1-josephfrazierhttpsgithubcomjosephfrazier-sosukesuzukihttpsgithubcomsosukesuzuki-thorn0httpsgithubcomthorn0-7516httpsgithubcomprettierprettierpull7516-by-bakkothttpsgithubcombakkot)

Reply:
HA! I just commented the opposite (https://github.com/emberjs/ember.js/pull/19154#discussion_r495614099). Sorry!

@BarryThePenguin - Would mind reverting the `avoid` change, and collapsing the non `--fix` changes to a single commit; then run `yarn eslint --fix .` and commit _just_ that in a second commit? I'll review the first (but trust `eslint --fix` to the right thing for the rest), then we can land...

Reply:
No worries 😅 That should be done now